tests/functional/avatar.js

```diff
@@ -100,28 +100,8 @@ define([
 
         .setFindTimeout(intern.config.pageLoadTimeout)
 
-        .findByCssSelector('.avatar-view')
-        .end()
-
         // this link should not be present on the page at all
```

Comment:
I think we can remove the `emailAvatarAb` stuff now, everyone has access to avatars since August 12. so 1-2 tests should go away